model_analyzer/triton/client/grpc_client.py

```diff
@@ -54,5 +54,4 @@ def get_model_config(self, model_name, num_retries):
         self.wait_for_model_ready(model_name, num_retries)
         model_config_dict = self._client.get_model_config(model_name,
                                                           as_json=True)
-        self.unload_model(model_name)
```

Comment:
Can we add an option to only disable it when the option is set to True? (e.g. the option could be named `allow_model_unloading`) Also, I think as we discussed this requires removing `unload_model` and `load_model` from `get_model_config`.

Reply:
`allow_model_unloading` would require the user to set it to false?  I think we want the default to be false for this new option, and if the user wants different behavior they set an option to true.  I think it should be called `model-unloading-disable` (I think disable-model-unloading is better, but for consistency with other options we should put the disable at the end of the name)

Reply:
`model-unloading-disable` sounds good to me. We need to also document that this option is only used during `remote` mode.

Reply:
@Tabrizian @tgerdesnv To handle both `unloading` and `loading`, how about renaming this to `model-loading-unloading-disable`?

Reply:
Red flag: Using this negative/not/disable flag name will yield code like this.
@tgerdesnv See the `not not` below?  I think we should reconsider the name of this flag.  It's a tradeoff between consistent (albeit bad) argument names, and clean code.  

```
            not_loading_and_unloading = analyzer_config['model_loading_unloading_disable']

            if not not_loading_and_unloading:
                self._client.load_model(model.model_name())
```

Reply:
I guess I need more context for this story/change. Surely we still have to initially load a model, right?  Is this story going to remove the **reloading** of the model?